tests/runner.py

```diff
@@ -70,6 +72,11 @@ def is_emterpreter(self):
 
   def setUp(self):
     Settings.reset()
+
+    for root, dirnames, filenames in os.walk(TEMP_DIR):
```

Comment:
i worry this will be very slow if the user has 1,000,000 files in the temp dir already. I know I sometimes do in `/tmp`...